vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -118,7 +122,7 @@ def fit(self,
                 # Recommended for applications that benefit from the best possible model accuracy.
                 best_quality={
                     'hyperparameters': {
-                        'model': Categorical('resnet50_v1b', 'resnet101_v1d', 'resnest200'),
+                        'model': Categorical('coat_lite_small', 'twins_pcpvt_base', 'swin_base_patch4_window7_224'),
```

Comment:
Why these models? Can we do a benchmark on which models actually give best quality? What about resnet101d and resnet101_v1d?

Reply:
@Innixma These are the cpp data result for different models, basically they have the same trend of large dataset(imagenet): https://github.com/rwightman/pytorch-image-models/blob/d5ed58d623be27aada78035d2a19e2854f8b6437/results/results-imagenet-real.csv
```csv
model,score_test,score_val,pred_time_test,pred_time_val,fit_time,pred_time_test_marginal  ,pred_time_val_marginal,fit_time_marginal,stack_level,can_infer,fit_order
1     swin_base_patch4_window7_224    0.940399   0.944386       14.661540      10.654264   764.274587                14.661540               10.654264         764.274587            1       True          4
2     twins_pcpvt_base    0.929423   0.937033       10.634345       7.909998   855.020386                10.634345                7.909998         855.020386            1       True          3
3     coat_lite_small    0.924517   0.930694        9.310690       7.005245   517.815660                 9.310690                7.005245         517.815660            1       True          2
4     gluon_resnet101_v1d    0.915675   0.939413        9.036937       6.305846   484.620409                 9.036937                6.305846         484.620409            1       True          1
```